scripts/run_validator_node.sh

```diff
@@ -0,0 +1,238 @@
+        echo "OK"
+    else
+        echo "FAIL"
+        echo "No systemd detected. Exiting." >&2
+        exit 1
+    fi
+fi
+
+if ! command -v jq &>/dev/null || \
+    ! command -v curl &>/dev/null
+then
+    echo "You need to install some dependencies before continuing:"
+    echo "  jq curl"
+    exit 1
+fi
+
+while [ "$chain_spec_name" == "" ]; do
+    echo "Available networks: "
+    echo "1) Devnet"
+    echo "2) Testnet"
+    echo "3) Mainnet"
+    read -p "Select network for your running node: " choice
+    case $choice in
+        1)
+            chain_spec_name="devnet"
+            ;;
+        2)
+            chain_spec_name="testnet"
+            ;;
+        3)
+            chain_spec_name="mainnet"
+            ;;
+        *)
+            echo "Invalid choice. Please select a right number."
+            continue
+            ;;
+    esac
+done
+
+echo "Your choice is: $chain_spec_name network."
+
+while [ -z "$files_path" ]; do
+    read -p "Enter the path where you want to store at least a few gigabytes of data (or press Enter to use the standard $HOME/atleta/chain directory): " files_path
+
+    if [ -z "$files_path" ]; then
+        files_path="$HOME/atleta/chain"
+        binary_path="/opt/atleta"
+        $sudo_cmd mkdir -p "$files_path"
+        $sudo_cmd chmod -R 777 "$files_path"
+        echo "Standard directory selected: $files_path"
+    else
+        files_path="$HOME/$files_path"
+        binary_path="/usr/local"
+        $sudo_cmd mkdir -p "$files_path"
+        $sudo_cmd chmod -R 777 "$files_path"
+    fi
+
+    if [ -w "$files_path" ]; then
+        echo "The directory exists and you have write permissions."
+    else
+        echo "You do not have write permission to the specified directory. Please select another directory."
+        files_path=""
+    fi
+done
+
+if [ "$(ls -A $files_path/chains &>/dev/null)" ]; then
+    keychain_exists=1
+else
+    keychain_exists=0
+fi
+
+echo "Everything's ready. Tasks:"
+echo "  [X] Download atleta-node -> $binary_path/bin/atleta-node"
+echo "  [X] Download $chain_spec_name network chain spec -> $binary_path/etc/chain_spec.$chain_spec_name.json"
+if [ $keychain_exists -eq 0 ]; then
+    echo "  [X] Generate new session keys and store them in node"
+else
+    echo "  [ ] Data dir already exists, so session keys won't be regenerated."
+fi
+echo "Press Enter to continue or Ctrl-C to cancel."
+read < /dev/tty
+
+echo "Create directories..."
+$sudo_cmd mkdir -p "$binary_path/bin" "$binary_path/etc"
+echo "Stop old node if it's running..."
+
+#Check status of process
+if [[ $operating_system == "Linux" ]]; then
+    $sudo_cmd systemctl stop atleta-validator &>/dev/null || true
+else
+    $sudo_cmd launchctl unload /Library/LaunchDaemons/com.example.atleta-validator.plist || true
+fi
+
+echo "Download binary..."
+$sudo_cmd curl -sSL https://github.com/Atleta-network/atleta/releases/download/v1.0.0/atleta-node -o "$binary_path/bin/atleta-node"
```

Comment:
These hardcoded URLs should be part of script header for easy editing and overview

Reply:
Oh, and just noticed you hard-coded the version. So in case we released 1.0.1, we would need to update the script as well